app.js

```diff
@@ -1,9 +1,25 @@
 const { getTopics } = require('./controllers/topics.controller');
+const { getArticleById } = require('./controllers/articles.controller');
+
 const express = require('express');
 const app = express();
 
 app.use(express.json());
 
 app.get('/api/topics', getTopics);
+app.get('/api/articles/:article_id', getArticleById);
+
+//Error handling
+app.use((err, req, res, next) => {
+  if (err.code === '22P02') {
+    res.status(400).send({ msg: 'Bad request!' });
+  } else next(err);
+});
+
+app.use((err, req, res, next) => {
+  if (err.status && err.msg) {
+    res.status(err.status).send({ msg: err.msg });
+  }
+});
```

Comment:
Great stuff, in the future you may want to consider putting these error handling middleware functions in their own file - but works well for now 😄